docs/source/extension/extension_dev.rst

```diff
@@ -548,7 +548,7 @@ While authoring a prebuilt extension, you can use the ``labextension develop`` c
 
    jupyter labextension develop . --overwrite
 
-Then rebuilding your extension and refreshing JupyterLab in the browser should pick up changes in your prebuilt extension source code.
+Then rebuilding your extension and refreshing JupyterLab in the browser should pick up changes in your prebuilt extension source code. If Windows is used, please, take a look how to configure Windows :ref:`important-for-windows-users`
```

Comment:
Three things here:
1. Splitting to new line to make editing easier (it will render the same)
2. Using active voice to align with the Jupyterlab documentation [writing style](https://jupyterlab.readthedocs.io/en/latest/developer/contributing.html#writing-style).
3. Explicitly mentioning that the `develop` command may not work without additional configuration.

```suggestion
Then rebuilding your extension and refreshing JupyterLab in the browser should pick up changes in your prebuilt extension source code.
If using Windows, you may need to configure your operating system for the ``develop`` command described above to work, please see the note: :ref:`important-for-windows-users`
```